### PR TITLE
Let layers be strings.

### PR DESCRIPTION
**Pull request by rodrigogiraoserrao**
Motivated by syntax highlighting of TCSS, I'd like us to consider allowing the values of `layer` and `layers` be strings.
I.e., allow CSS like this:

```sass
Screen {
    layers: "layer-name";
}

#widget {
    layer: "layer-name";
}
```

Nothing needs to be done for `layer` because it doesn't check its token type.
For `layers`, the change is trivial and can be seen in this PR.

### CODE REVIEW

File: src/textual/css/_styles_builder.py [hunk 1]
```diff
@@ -722,7 +722,7 @@ def process_layer(self, name: str, tokens: list[Token]) -> None:
     def process_layers(self, name: str, tokens: list[Token]) -> None:
         layers: list[str] = []
         for token in tokens:
-            if token.name != "token":
+            if token.name not in ["token", "string"]:
```
Comment (willmcgugan) [inline src/textual/css/_styles_builder.py:725]: You could save a few microseconds with a set literal.

**Comment by willmcgugan** [comment]
Seems reasonable. Could you add a changelog entry?

**Comment by rodrigogiraoserrao** [comment]
> Seems reasonable.

Just to confirm: seems reasonable that we accept strings and push it as the "standard" or just that we have this possibility?

**Comment by willmcgugan** [comment]
I think strings are only necessary if you want additional characters or spaces in your layer names. I wouldn't say it should be the standard exactly, just an alternative.

**Review by willmcgugan — APPROVED** [review]

**Pull request merged**